proto/profile.proto

```diff
@@ -89,6 +89,11 @@ message Profile {
   // Index into the string table of the type of the preferred sample
   // value. If unset, clients should default to the last sample value.
   int64 default_sample_type = 14;
+  // Index into the string table that defines the resolution of
+  // Breakdown.ticks, e.g. "nanoseconds" or "microseconds".
+  int64 tick_unit = 15;
```

Comment:
Given that above we have `time_nanos` and `duration_nanos` is there a realistic need to increase complexity here instead of just making all ticks in nanos too?

Reply:
It adds complexity, but the space savings are significant. I'll provide more data on this, but in my initial testing going from `ns` to `us` to `ms` resolution results in 20% file size reduction each time.

The pprof format also already recognizes (time) units as first class citizens, see `profile.proto` and [`measurement.go`](https://github.com/google/pprof/blob/70bd9ae97f40a5e0ef03e61bec8e90f3a5732191/internal/measurement/measurement.go#L261-L293).